test/request-response-samples/Dummy__should_return_array_despite_maxOccurs_1/response.xml

```diff
@@ -0,0 +1,20 @@
+<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:tns="http://www.Dummy.com" xmlns:n="http://www.Dummy.com/Name/Types" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <soap:Header></soap:Header>
+    <soap:Body>
+       <n:DummyResponse>
+           <n:DummyResult>
+               <c:DummyList xmlns:c="http://www.Dummy.com/Common/Types">
+                   <c:DummyElement>
```

Comment:
is this already handled , i.e, returning an array even for an element with maxoccurs 1, if multiple occurrences are found ?

Reply:
are we adding this test case for regression or is this test for the new proposed feature

Reply:
This is a test case for regression. The package as published already exhibits this behaviour.